CHANGELOG

```diff
@@ -1,3 +1,23 @@
+Version 0.4.0 (2018-07-16)
+--------------------------
+Migrate from joda.time to java.time (#72)
+Create caching Dark Sky client (#73)
+Integrate Dark Sky as a weather provider (#42)
+Split Client into api-specific Clients and agnostic Transport (#71)
+Include configuration errors in the error hierarchy (#67)
+Make WeatherError extend Exception (#68)
+Make all case classes final (#69)
+Migrate from twitter-util-collection to scala-lru-map (#46)
+Rename OwmAsyncClient to OwmClient (#65)
+Make constructQuery typesafe (#62)
+Remove ValidatedWeather type alias (#61)
+Wrap OwmCacheClient operations in effect type (#60)
+Migrate from json4s to circe (#44)
+Replace scalaj with Hammock (#45)
+Bump Joda-Time versions (#58)
+Bump Scala versions to 2.11.12 and 2.12.6 (#53)
+Remove Vagrant setup (#43)
```

Comment:
it seems there are 32 issues in https://github.com/snowplow/scala-weather/issues?q=is%3Aissue+milestone%3A%22Version+0.4.0%22

Reply:
I removed those less relevant to users, but I will add it back :+1: